app/components-react/windows/settings/Multistreaming.tsx

```diff
@@ -55,24 +55,26 @@ export function MultistreamingSettings() {
               </li>
               <li>
                 <Translate
+                  // ignore \" for intl translation
+                  // prettier-ignore
                   message={
-                    'Step 2: Ensure the "Confirm stream title and game before going live" option is checked in the <general>General</general> settings tab."'
+                    'Step 2: Ensure the \"Confirm stream title and game before going live\" option is checked in the <general>General</general> settings tab."'
                   }
                 >
                   <u slot="general" />
                 </Translate>
               </li>
               <li>
-                {$t('Step 3: Select which platforms you are streaming to when you hit "Go Live".')}
+                {
+                  // ignore \" for intl translation
+                  // prettier-ignore
+                  $t('Step 3: Select which platforms you are streaming to when you hit \"Go Live\".')
+                }
```

Comment:
i'm not really sure what the point of this tab is anymore 🤔 weren't there supposed to be some dual output settings here?